app/code/Magento/InventoryInStorePickup/Model/Carrier/Command/GetFreeBoxes.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryInStorePickup\Model\Carrier\Command;
```

Comment:
Do we really want shipping method right inside the `InStorePickup` module?

Reply:
Moved to separated module.